onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -30,7 +30,7 @@ OpenVINOExecutionProvider::OpenVINOExecutionProvider(const OpenVINOExecutionProv
   //to check if target device is available
   //using ie_core capability GetAvailableDevices to fetch list of devices plugged in
   bool device_found = false;
-  auto available_devices = openvino_ep::BackendManager::GetGlobalContext().ie_core.GetAvailableDevices();
+  auto available_devices = openvino_ep::BackendManager::GetGlobalContext().ie_core.get_available_devices();
```

Comment:
get_available_devices() is not available with 2021.4, it will throw an error there.

Till 2021.4, it should be ie_core.GetAvailableDevices() and then ie_core.get_available_devices() starting from 2022.1